main.tf

```diff
@@ -0,0 +1,16 @@
+data "template_file" "default" {
+  template = "${file("${var.ssh_public_key_path}")}"
+}
+
+
+module "label" {
+  source    = "git::https://github.com/cloudposse/tf_label.git?ref=tags/0.2.0"
```

Comment:
Add support of all `tf_label` `0.2.0` capabilities.

Reply:
Apply `terraform plan`.

Reply:
has been fixed